package.json

```diff
@@ -74,6 +74,7 @@
   },
   "webextensionManifest": {
     "manifest_version": 2,
+    "name": "Snooze Tabs",
```

Comment:
Should/can this be localized, too, now that we're not using the one from the outer scope?

Reply:
I think we've been avoiding localizing product / project names in general